starknet_py/net/models/transaction_test.py

```diff
@@ -103,6 +109,21 @@ def test_declare_compress_program(balance_contract):
             ),
             1691558101504686217378182149804732367606605343820187119932616442583251634573,
         ),
+        (
+            DeclareV2(
+                contract_class=create_new_compiled_contract(
+                    compiled_contract=new_compiled_contract
+                ),
+                compiled_class_hash=0x1,
+                max_fee=1000,
+                nonce=20,
+                sender_address=0x1234,
+                signature=[0x1, 0x2],
+                version=2,
+            ),
+            # FIXME: This may not be correct compared to starknet calculations
+            88310833614671781001635391408220089034532366759635809827057930536492581034,
```

Comment:
Because the value itself comes from our calculations. But it seems that sending declare v2 from starknet.py is working so the class hash must be correct I think.